src/articles/articles.controller.ts

```diff
@@ -78,6 +102,19 @@ export class ArticlesController extends BaseController implements IArticlesContr
 		}
 	}
 
+	async publishArticle(req: Request, res: Response, next: NextFunction): Promise<void> {
+		try {
+			const id = req.params.id;
+			const { isPublished } = req.body;
+
+			const result = await this.articleService.publishArticle(id, isPublished);
+
+			this.ok(res, result);
+		} catch (e) {
+			return next(e);
+		}
+	}
+
 	async deleteArticle({ params }: Request, res: Response, next: NextFunction): Promise<void> {
 		try {
 			const id = params.id;
```

Comment:
if manager didn't create this article then he can't delete it

Reply:
Done